cli/dcoscli/marathon/main.py

```diff
@@ -364,22 +364,27 @@ def _add(app_resource):
     :returns: process return code
     :rtype: int
     """
-    application_resource = _get_resource(app_resource)
+    application_resources = _get_resource(app_resource)
 
     # Add application to marathon
     client = marathon.create_client()
 
     # Check that the application doesn't exist
-    app_id = client.normalize_app_id(application_resource['id'])
+    if not isinstance(application_resources, list):
+        application_resources = [application_resources]
 
-    try:
-        client.get_app(app_id)
-    except DCOSException as e:
-        logger.exception(e)
-    else:
-        raise DCOSException("Application '{}' already exists".format(app_id))
+    for application_resource in application_resources:
+        app_id = client.normalize_app_id(application_resource['id'])
+
+        try:
+            client.get_app(app_id)
+        except DCOSException as e:
+            logger.exception(e)
+        else:
+            raise DCOSException("Application '{}' already exists".format(
```

Comment:
This will end execution. Do we want to try all the apps and add all the ones that don't already exist?